src/frontend/utils/conversejs/converse-plugins/marshaJoinDiscussionPlugin.ts

```diff
@@ -58,9 +63,11 @@ const addMarshaJoinDiscussionPlugin = (xmpp: XMPP) =>
             ) {
               const jid = messageStanza.getAttribute('from');
               const username = converse.env.Strophe.getResourceFromJid(jid);
-              useJoinParticipant
-                .getState()
-                .addParticipantAskingToJoin({ id: jid, name: username });
+              const video = useVideo.getState().getVideo(appData.video!);
```

Comment:
I have quitte an ambigus feeling about the current use of video from appData, from the hook useVideo and from props.

Reply:
Can you explain this please ?

Reply:
How can I get the video in a converse plugin ?